telemetry/generated/element.ts

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// AUTOGENERATED BY glean_parser v7.2.1. DO NOT EDIT. DO NOT COMMIT.
```

Comment:
We recommend to not vendor the generated files, but rather generate them as part of the deployment process (for a variety of reasons, including playing nice with local linters and the fact that they are not necessarily stable across glean updates). Just a recommendation though: your call!